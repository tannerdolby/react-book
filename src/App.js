import BookList from './components/BookList';

function App() {
  return (
    <div className='app'>
      <div className='book-demo'>
        <BookList />
      </div>
    </div>
  );
}

export default App;
