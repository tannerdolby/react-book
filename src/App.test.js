import { render } from '@testing-library/react';
import App from './App';

describe('renders app', () => {
  it('matches snapshot', () => {
    const component = render(<App />);
    expect(component.container).toMatchSnapshot();
  });
});
