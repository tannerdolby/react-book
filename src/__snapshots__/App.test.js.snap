// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app matches snapshot 1`] = `
<div>
  <div
    class="app"
  >
    <div
      class="book-demo"
    >
      <button
        class="next-btn"
      >
        View Next Cover
      </button>
      <div
        class="book cube"
        tabindex="0"
        title="The Hitch Hiker's Guide to the Galaxy"
      >
        <div
          class="cube front"
        />
        <div
          class="cube back"
        />
        <div
          class="cube right"
        />
        <div
          class="cube left"
        />
        <div
          class="cube top"
          style="background-image: url(undefined);"
        />
        <div
          class="cube bottom"
        />
        <div
          class="cube shadow"
        />
      </div>
    </div>
  </div>
</div>
`;
